systemtest/src/test/java/io/strimzi/systemtest/operators/user/UserST.java

```diff
@@ -403,8 +407,9 @@ void testTlsExternalUser(ExtensionContext extensionContext) {
      *  - secrets
      */
     @ParallelTest
-    @KRaftNotSupported("Probably bug in Kafka - https://issues.apache.org/jira/browse/KAFKA-13964 with fix available in kafka 3.5.0")
     void testUOListeningOnlyUsersInSameCluster(ExtensionContext extensionContext) {
+        assumeTrue(Environment.isKRaftModeEnabled() && TestKafkaVersion.compareDottedVersions("3.5.0", Environment.ST_KAFKA_VERSION) != 1);
```

Comment:
Should you open some issue to remove all of these once we drop support for Kafka 3.4.x?

Reply:
Yeah, I wanted to create one, thanks